.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E226,E302,E41
+max-line-length = 90
+exclude = tests/*
```

Comment:
A good practice is to exclude venv/* and migrations/* as well.

Reply:
I see.